setup.py

```diff
@@ -67,6 +67,7 @@ def get_version():
               'wheel==0.24.0',
               'moto==0.4.22',
               'flake8==2.5.0',
+              'freezegun==0.3.9',
```

Comment:
Ooooh, cool!!

Reply:
Yeah. Unfortunately it doesn't work on my timeout-style tests.